### PR TITLE
fix: fixed websocket multiple response issue

### PR DESCRIPTION
**Pull request by Astitva877**
## Description

Added a websocket listener store to store websocket listener and remove it after disconnect 
#1687 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: src/packages/@workspaces/features/socket-explorer/store/websocket-listener.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import type { UnlistenFn } from "@tauri-apps/api/event";
+import { writable } from "svelte/store";
+
+export const webSocketListener = writable<Map<string, UnlistenFn>>(new Map());
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/socket-explorer/store/websocket-listener.ts:4]: no need to create a new store, already handling  a hashmap based store for web socket with corresponding to tabId. that holds tab related data at one place

File: src/packages/@deprecate/api/api.common.ts [hunk 2]
```diff
@@ -14,7 +14,10 @@ import { Events } from "$lib/utils/enums/mixpanel-events.enum";
 import type { MakeRequestResponse } from "$lib/utils/interfaces/common.interface";
 import type { Response } from "$lib/utils/interfaces/request.interface";
 import { listen } from "@tauri-apps/api/event";
-import { webSocketDataStore } from "@workspaces/features/socket-explorer/store";
+import {
+  webSocketDataStore,
```
Comment (itsmdasifraza) [inline src/packages/@deprecate/api/api.common.ts:18]: we can use this one only

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**